tests/test_coverage.py

```diff
@@ -58,3 +58,27 @@ def test_sum():
     result.stdout.fnmatch_lines([
         'lib.py*4*0*100%'
     ])
+
+
+def test_unsupported_lanuage(testdir):
+    # Setup notebook with one empty cell
+    nb = build_nb([''])
+    # An unsupported language name (please do not call your language this (-_-*))
+    nb.metadata = {'language_info': {'name': 'myCustomLanguage++'}}
+
+    # Write notebook to test dir
+    nbformat.write(nb, os.path.join(
+        str(testdir.tmpdir), 'test_unupported_coverage.ipynb'))
+
+    # Run tests
+    result = testdir.runpytest_inprocess('--nbval', '--current-env', '--cov', '.')
+
+    # Check tests went off as they should
+    # (test will have errors due to mismatching kernel):
+    result.ret == 0
```

Comment:
This presumably isn't doing anything without an `assert`?

Reply:
I ended up removing the test, as there were several things wrong with it (I think that we would have to mock a valid kernel for the test).